app/globals.css

```
@@ -81,3 +81,21 @@ main {
 .dark .prose :not(pre) > code::before {
   color: rgb(var(--color-accent-strong));
 }
+
+.aurora {
+  background:
+    radial-gradient(circle at 10% 30%, rgb(var(--color-accent) / 0.34), transparent 35%),
```

Stylelint (17.2.0)

[error] Expected "0.34" to be "34%" (alpha-value-notation)

(alpha-value-notation)